.github/workflows/new-issue.yml

```diff
@@ -9,21 +9,106 @@ name: "new-issue"
+      PLUGIN_VERSION: ${{ steps.missed_plugin_version.outputs.PLUGIN_VERSION }}
+      OS_VERSION: ${{ steps.missed_os_version.outputs.OS_VERSION }}
+    steps:
+      - uses: actions/checkout@v2.0.0 
+      - name: Missed Steps to reproduce
+        id: steps_to_reproduce
+        if: ${{contains(github.event.issue.body, 'Steps To Reproduce') == false}}
+        env:
+          STEPS_REPRODUCE: Steps to reproduce
+        run: echo "::set-output name=STEPS_REPRODUCE::$STEPS_REPRODUCE"
+      - name: Missed Link to a GitHub repository
+        id: link_to_github
+        if: contains(github.event.issue.body, 'Repository to reproduce') == false
+        env:
+          LINK_TO_GITHUB: Link to a GitHub repository
+        run: echo "::set-output name=LINK_TO_GITHUB::$LINK_TO_GITHUB"
+      - name: Missed Expected result
+        id: expected_result
+        if: contains(github.event.issue.body, 'Expected result') == false
+        env:
+          EXPECTED_RESULT: Expected result
+        run: echo "::set-output name=EXPECTED_RESULT::$EXPECTED_RESULT"
+      - name: Missed Actual result
+        id: actual_result
+        if: contains(github.event.issue.body, 'Actual result') == false
+        env:
+          ACTUAL_RESULT: Actual result
+        run: echo "::set-output name=ACTUAL_RESULT::$ACTUAL_RESULT"
+      - name: Missed CLI version
+        id: missed_cli_version
+        if: contains(github.event.issue.body, 'CLI Version') == false
+        env:
+          CLI_VERSION: CLI version
+        run: echo "::set-output name=CLI_VERSION::$CLI_VERSION"
+      - name: Missed plugin version
+        id: missed_plugin_version
+        if: contains(github.event.issue.body, 'plugin Version') == false
+        env:
+          PLUGIN_VERSION: Plugin version
+        run: echo "::set-output name=PLUGIN_VERSION::$PLUGIN_VERSION"
+      - name: Missed OS version
+        id: missed_os_version
+        if: contains(github.event.issue.body, 'OS') == false
+        env:
+          OS_VERSION: OS version
+        run: echo "::set-output name=OS_VERSION::$OS_VERSION"
+  display-message:
+    runs-on: ubuntu-latest
+    needs: [triage, missed-info]
+    steps:
+      - uses: actions/checkout@v2.0.0 
+      - name: issue comment
+        if: ${{needs.missed-info.outputs.STEPS_REPRODUCE != '' 
+          || needs.missed-info.outputs.LINK_TO_GITHUB != ''
+          || needs.missed-info.outputs.EXPECTED_RESULT != ''
+          || needs.missed-info.outputs.ACTUAL_RESULT != ''
+          || needs.missed-info.outputs.CLI_VERSION != ''
+          || needs.missed-info.outputs.PLUGIN_VERSION != ''
+          || needs.missed-info.outputs.OS_VERSION != ''}}
+        id: issue_comment_missing
+        uses: ./
+        with: 
+          repo-token: ${{ secrets.GITHUB_TOKEN}}
+          message: >
+            ${{format('Your issue is missing this required information: {0} {1} {2} {3} {4} {5} {6}. When you provide us this information we''ll triage and reproduce the issue.',
```

Comment:
This doesn't quite work this way. Negative lookups only return one right? For example:

```javascript
'Steps To Reproduce\nRepositoryto reproduce\nExpected reslt\nActual result\nCLI Version\nplugin Version\nOS'.match(/^((?!Steps To Reproduce)[\s\S])*$|^((?!Repository to reproduce)[\s\S])*$|^((?!Expected result)[\s\S])*$|^((?!Actual result)[\s\S])*$|^((?!CLI Version)[\s\S])*$|^((?!plugin Version)[\s\S])*$|^((?!OS)[\s\S])*$/)
[
  'Steps To Reproduce\n' +
    'Repositoryto reproduce\n' +
    'Expected reslt\n' +
    'Actual result\n' +
    'CLI Version\n' +
    'plugin Version\n' +
    'OS',
  undefined,
  'S',
  undefined,
  undefined,
  undefined,
  undefined,
  undefined,
  index: 0,
  input: 'Steps To Reproduce\n' +
    'Repositoryto reproduce\n' +
    'Expected reslt\n' +
    'Actual result\n' +
    'CLI Version\n' +
    'plugin Version\n' +
    'OS',
  groups: undefined
]
```
As you can see, it is only marking the second when there are other sections missing. If we want this level of complexity, I'm not sure doing it in regexp and yaml is the way to do it.